README.md

```diff
@@ -47,32 +47,74 @@ applications running in almost any environment. Here's an introductory video:
 
   Open Stackdriver Error Reporting at https://console.cloud.google.com/errors to view the reported errors.
 
-## Setup
+## Running on Google Cloud Platform
 
-When initing the Stackdriver Error Reporting library you must specify the following:
+There are three different services that can host Node.js application to Google Cloud Platform.
```

Comment:
nit: I am not sure there is much utility of this sentence – it just states the number of subsections that follow. I think you should just skip this.